### ISSUE TITLE
Apple Pay doesnt work

### ISSUE THREAD

**Issue by mickadoua** [mkharibalaji/react-native-adyen-payment#55]
Hi, 
I try since 2 days to add apple pay to my application

Here what i did, 

```

const computeComponentData = {
   ...
  applepay: {
    apple_pay_merchant_id: 'xxx',
  }, 
}


 return AdyenPayment.startPaymentPromise(
    paymentMethod: AdyenPayment.APPLE_PAY,
    computeComponentData(),
    computePaymentDetails({
      amount,
      shopperEmail,
      shopperReference,
    })
  );
```

my proxy recieve strange information 

```
{
    "amount": {
        "value": 1500,
        "currency": "EUR"
    },
    "channel": "iOS",
    "storePaymentMethod": false,
    "reference": "xxx",
    "shopperReference": "xxx",
    "shopperEmail": "xxx@xxxx.com",
    "returnUrl": "fr.xxxx.ios.preprod:\/\/",
    "countryCode": "FR",
    "paymentMethod": {
        "applePayCardNetwork": "Visa",
        "applepay.token": "", <=== 
        "type": "applepay"
    },
    "shopperLocale": "fr_FR",
    "additionalData": {
        "allow3DS2": true,
        "executeThreeD": true
    },
    "merchantAccount": "FR_ZEturf-ZEbet"
}

```

on [adyen v51 api's documentation](https://docs.adyen.com/api-explorer/#/CheckoutService/v51/post/payments__reqParam_paymentMethod-ApplePayDetails-applePayToken), "applepay.token" variable  should be "applePayToken" and of course, the value shouldnt be empty

**Comment by ChielBruin** [comment]
See https://github.com/Adyen/adyen-ios/issues/204
TLDR: you cannot use a simulator to tests Apple Pay, you need to set up an iPhone with a Apple Sandbox account in order to test it

**Comment by mickadoua** [comment]
thanks @ChielBruin, i didn't check issue on adyen/ios
thanks you,

**Comment by ChielBruin** [comment]
We should probably add this to the docs. I was stuck on this for while too

**Comment by mickadoua** [comment]
Yes its good idea to add this to the documentation 

BTW did you follow [this step from Adyen documentation](https://docs.adyen.com/payment-methods/apple-pay/enable-apple-pay) to ask a certificat from adyen ?   i still cant figure why i get an error from adyen Api
